msgpack/fallback.py

```diff
@@ -149,6 +149,42 @@ def _unpack_from(f, b, o=0):
     _unpack_from = struct.unpack_from
 
 
+MessagePack = namedtuple("MessagePack", ["size", "format", "type"])
+MessagePack.__new__.__defaults__ = ("", -1)
```

Comment:
I don't like namedtuple here.
Use simple tuple, and `size, fmt, type = _MSGPACK_HEADERS[b]` instead.